app/Model/Cert.php

```diff
@@ -491,4 +491,39 @@ public function beforeSave($options = array())
     return true;
   }
 
+  /**
+   * afterSave
+   *
+   * @param  boolean true if a new record was created (rather than update)
+   * @param  array, the same passed into Model::save()
+   * @return none
+   */
+  function afterSave($created, $options = Array()) {
+    $this->manualProvisionCert();
+  }
+
+  /**
+   * afterDelete
+   *
+   * @return none
+   */
+  function afterDelete() {
+    $this->manualProvisionCert();
+  }
+  
+  /**
+   * manualProvisionCert
+   *
+   * @return none
+   */
+  function manualProvisionCert() {
+    $sModel = 'CoPerson';
```

Comment:
Why you hardcoded the Model Name? If you need the CoPerson Model change this
```php
   $this->$sModel->Behaviors->load('Provisioner');
    /* provision CoPerson for Cert*/
    $this->$sModel->manualProvision(null, $sId, null, $sAction);
```
to this
```
   $this->CoPerson->Behaviors->load('Provisioner');
    /* provision CoPerson for Cert*/
    $this->CoPerson->manualProvision(null, $sId, null, $sAction);
```